docs/cli-guide-mtr/master.adoc

```diff
@@ -1,4 +1,4 @@
-:mtr:
+:{LC_PSN}:
 include::topics/templates/document-attributes.adoc[]
 :_content-type: ASSEMBLY
 [id="cli-guide"]
```

Comment:
same question as above -- leave attribute alone. The substitutions are just for the text.